### ISSUE TITLE
[Intentional or bug?] v4.3.19 `brew list` now outputs single column lists

### ISSUE THREAD

**Issue by dalDevelo** [Homebrew/brew#18232]
### `brew doctor` output

```shell
➜  brew doctor
Your system is ready to brew.
```

### Verification

- [X] My "`brew doctor` output" above says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` twice and am still able to reproduce my issue.
- [X] This issue's title and/or description do not reference a single formula e.g. `brew install wget`. If they do, open an issue at https://github.com/Homebrew/homebrew-core/issues/new/choose instead.

### `brew config` output

```shell
➜  brew config
HOMEBREW_VERSION: 4.3.19
ORIGIN: https://github.com/Homebrew/brew
HEAD: 1f9bd2de89883ec9a59db3eab0f5475bce06d5bc
Last commit: 12 hours ago
Core tap JSON: 02 Sep 21:09 UTC
Core cask tap JSON: 02 Sep 21:09 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.6.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
Rosetta 2: false
```

### What were you trying to do (and why)?

Use `brew list` to view installed formulae and casks

### What happened (include all command output)?

`brew list` shows single column lists since version 4.3.19

<img width="682" alt="Screenshot 2024-09-02 at 22 24 35" src="https://github.com/user-attachments/assets/35df917b-3503-47d1-b1af-ee9e4b261f96">

### What did you expect to happen?

Expected to see multi-column lists as displayed with `brew ls`

<img width="682" alt="Screenshot 2024-09-02 at 22 24 42" src="https://github.com/user-attachments/assets/5dc99b73-5148-4ed1-a9b4-ff7653d9b75d">

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
Run `brew list`
Run `brew ls`
```

**Comment by Bo98** [comment]
Not intentional specifically, though we did change how it worked (it's now significantly faster). I think I can fix the output to be similar to before however.

**Comment by dalDevelo** [comment]
It's not the end of the world if it remains this way, but Including a small prompt to use `brew ls` for multi column view could be useful.

**Comment by Bo98** [comment]
`brew ls` being different is also not intentional either. We'll fix and align this in 4.3.20.
